diff --git a/app/gui/qt/CMakeLists.txt b/app/gui/qt/CMakeLists.txt
index 52056559d..0127b5fd9 100644
--- a/app/gui/qt/CMakeLists.txt
+++ b/app/gui/qt/CMakeLists.txt
@@ -282,7 +282,6 @@ set_property(TARGET ${APP_NAME} PROPERTY POSITION_INDEPENDENT_CODE ON) # Qt requ
 target_include_directories(${APP_NAME}
     PRIVATE
     ${QTAPP_ROOT}
-    ${APP_ROOT}/external/scsynth-boost-1.74.0
     ${APP_ROOT}/external/TLSF-2.4.6/src
     ${APP_ROOT}/external
     ${QTAPP_ROOT}/osc
